base/docs/helpdb/Base.jl

```diff
@@ -666,9 +666,9 @@ references in `r` are replaced with the corresponding matched text.
 replace
 
 """
-    chop(string)
+    chop(string, n = 1)
 
-Remove the last character from a string.
+Remove the last `n` characters from a string.
```

Comment:
To be clearer that this is not mutating the string, maybe
```
Returns a string with the last `n` characters removed.  If `n` is greater than the length of the string, returns an empty string.
```

If `n` is greater than the number of characters in the string, retur